app/src/main/java/com/github/se/cyrcle/model/map/MapViewModel.kt

```diff
@@ -186,6 +232,72 @@ class MapViewModel : ViewModel() {
     }
   }
 
+  /**
+   * Draw markers on the map.
+   *
+   * @param pointAnnotationManager the PointAnnotationManager to draw the markers on
+   * @param parkingList the list of parkings to draw
+   * @param bitmap the bitmap to use for the markers
+   */
+  fun drawMarkers(
```

Comment:
If we're strictly MVVM here... a draw function takes data and puts it into the UI, which is exactly the purpose of a ViewModel in my opinion. It has its place here

Reply:
Since we are manipulating the `pointAnnotationManager` and the data itself for me it should belong in the view model. I don't think there is a problem since this way the view model can control what data is displayed on the map.